packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -32,54 +29,53 @@ const Dashboard = ({ active_tab, setActiveTab, toggleStrategyModal, is_drawer_op
     };
 
     return (
-        <div className='dashboard__main'>
-            <div className={classNames('dashboard__container', { 'w-100': !show_side_bar })}>
-                <ReactJoyride steps={DBOT_ONBOARDING} run={tour_run} continuous={true} showProgress={true} />
-                <Tabs active_index={active_tab} onTabItemClick={setActiveTab} top>
-                    {/* [Todo] needs to update tabs comIcDashBoardComponentsTabponent children instead of using label property */}
-                    <Tab icon='IcDashboardComponentTab' label={localize('Dashboard')}>
-                        <DashboardComponents />
-                    </Tab>
-                    <Tab icon='IcBotbuilderTabIcon' label={localize('Bot Builder')} id='id-bot-builder'>
-                        <BotBuilder />
-                    </Tab>
-                    <Tab
-                        icon='IcQuickStrategyIcon'
-                        label={localize('Quick Strategy')}
-                        id='id-quick-strategy'
-                        onTabItemClick={handleClick}
-                    >
+        <>
+            <div className='dashboard__main'>
+                <div className='dashboard__container'>
+                    <ReactJoyride steps={DBOT_ONBOARDING} run={tour_run} continuous={true} showProgress={true} />
+                    <Tabs active_index={active_tab} onTabItemClick={setActiveTab} top>
+                        {/* [Todo] needs to update tabs comIcDashBoardComponentsTabponent children instead of using label property */}
+                        <div icon='IcDashboardComponentTab' label={localize('Dashboard')}>
+                            <DashboardComponent />
+                        </div>
+                        <div icon='IcBotBuilderTabIcon' label={localize('Bot Builder')} id='id-bot-builder' />
                         <div
-                            className={classNames('quick-strategy', {
-                                'quick-strategy__notifications-container--open': !is_drawer_open,
-                            })}
+                            icon='IcQuickStrategyIcon'
+                            label={localize('Quick Strategy')}
+                            id='id-quick-strategy'
```

Comment:
can we remove `id` from the beginning of id=?

Reply:
it will be taken care by @maryia-binary in separate PR.